ds-prep-python-variables/main.py

```diff
@@ -0,0 +1,36 @@
+first_name = 'James'
+last_name = "Nakano"
+full_name = first_name + " " + last_name
+print(full_name)
+
+height_in_inches = 69
+print(height_in_inches, type(height_in_inches))
+height_in_inches_float = float(height_in_inches)
+print(height_in_inches_float, type(height_in_inches_float))
+height_in_meters = 2.54 * height_in_inches_float/100
+print(height_in_meters)
+
+eats_plants = True
+eats_animals = False
+if eats_plants == True or eats_animals == True:
+  is_animal = True
+else:
+  is_animal = False
+if eats_plants == True and eats_animals == True:
+  is_omnivore = True
+else:
+  is_omnivore = False
+is_plant = not is_animal
+print(is_animal, is_omnivore, is_plant)
+
+mean_height_in_meters = 1.7155
+short_threshold_in_meters = 1.576
+tall_threshold_in_meters = 1.860
+is_mean_height = height_in_meters == mean_height_in_meters
+is_short = height_in_meters < short_threshold_in_meters
+is_tall = height_in_meters > tall_threshold_in_meters
```

Comment:
`is_tall` should also be `True` if `height_in_meters` is equal to `tall_threshold_in_meters`.

---

![image](https://user-images.githubusercontent.com/16143771/138183307-57be334a-0475-4ad8-8a76-c66b35c045c8.png)